front-end-web/src/Components/Pages/ExportPage.tsx

```diff
@@ -1,16 +1,33 @@
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { Button, TextField } from "@material-ui/core";
 
 /**
  * TODO: Add Documentation
  */
+let calendarName: string = "";
+
 const ExportPage: React.FunctionComponent<Props> = ({
   state,
   handlers,
   classes
 }) => {
   return (
     <Fragment>
+      <TextField
+        variant="standard"
+        label="Calendar name"
+        onChange={e => (calendarName = e.target.value)}
+      ></TextField>
+      <Button
+        variant="contained"
```

Comment:
Can you please add the attributes `color="primary` and `size="large"`? Otherwise, I think you're good to go! Nice work!